python/cudf_polars/cudf_polars/dsl/expr.py

```diff
@@ -936,7 +959,9 @@ class UnaryFunction(Expr):
             "cum_sum",
         }
     )
-    _supported_fns = frozenset(_supported_misc_fns.union(_supported_cum_aggs))
+    _supported_fns = frozenset(
+        _supported_misc_fns.union(_supported_cum_aggs).union(frozenset(_MAPPING.keys()))
```

Comment:
```suggestion
        _supported_misc_fns | _supported_cum_aggs | _MAPPING.keys()
```

(if it works).

If not:
```
frozenset().union(_supported_misc_fns, _supported_cum_aggs, _MAPPING.keys())
```
which definitely does.